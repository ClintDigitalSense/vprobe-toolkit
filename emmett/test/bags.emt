# simple test of bag syntax
bag b[12];
Start  b[VCPUID] <- TSC;
Finish a[VCPUID] <- TSC - b[VCPUID];
