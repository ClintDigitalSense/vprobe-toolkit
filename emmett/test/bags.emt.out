(version 0.2)
(defaggr a 1 0)
(defbag b 12)

 ;;;;;
(vprobe Start ( baginsert b VCPUID TSC))
 ;;;;;
(vprobe Finish ( aggr  a ( VCPUID) ()  (- TSC  (bagremove b VCPUID) )))